AutomatedTesting/Gem/PythonTests/Performance/tests/EditorLevelLoading_base.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
This appears to be a "perf_timer" util instead of a "EditorLevelLoading_base"

Reply:
That's fair, will rename